app/frontend/src/views/merge/MergeAbout.vue

```diff
@@ -20,27 +20,25 @@
             &quot;General&quot;), or convert your sheet to CSV and format your dates to &quot;yyyy-mm-dd&quot; and time
             stamps to &quot;yyyy-mm-dd h:mm&quot;.
           </p>
-
           <ul class="mb-4">
             <li>
-              <a :href="`${publicPath}examples/mssc-ches-merge-example.csv`" download="mssc-ches-merge-example-csv.csv">example csv</a>
+              <a :href="`${publicPath}examples/mssc-ches-merge-example.csv`" download="mssc-ches-merge-example-csv.csv">example data csv</a>
             </li>
             <li>
-              <a :href="`${publicPath}examples/mssc-ches-merge-example.xlsx`" download="mssc-ches-merge-example-excel.xlsx">example xlsx</a>
+              <a :href="`${publicPath}examples/mssc-ches-merge-example.xlsx`" download="mssc-ches-merge-example-excel.xlsx">example data xlsx</a>
```

Comment:
Optional: May be better to have it be `example csv data` and `example xslx data` instead (order inversion).